examples/nuxt-langchain/.env.example

```diff
@@ -1 +1 @@
-OPENAI_API_KEY=
+NUXT_OPENAI_API_KEY=
```

Comment:
is this being used?

Reply:
Yes, this is consumed by Nuxt runtime config.

Reply:
ah very cool, I didn't realize that's how useRuntimeConfig worked.